addon/services/in-viewport.js

```diff
@@ -85,11 +85,9 @@ export default class InViewport extends Service {
           return;
         }
 
-        // grab the user added callbacks when we enter/leave the element
-        const {
-          enterCallback = noop,
-          exitCallback = noop
-        } = this.getCallbacks(element) || {};
```

Comment:
@gmurphey Looks like this is perfectly reasonable!   Mind removing the `getCallbacks` method below as well as the one in `/-private/raf-admin`?

Reply:
Thanks, @snewcomer! Should be all set.